libraries/resource_providers/api300/c7000/logical_interconnect_group_provider.rb

```diff
@@ -16,6 +16,9 @@ module API300
     module C7000
       # LogicalInterconnectGroup API300 C7000 provider
       class LogicalInterconnectGroupProvider < API200::LogicalInterconnectGroupProvider
+        # TODO: Inherit from API300::Synergy::LogicalInterconnectGroupProvider once the SDK
```

Comment:
I'm think it's good to go. I would just recommend opening an issue for this so we don't lose track of it.